components/event-publisher-proxy/pkg/env/types.go

```diff
@@ -1,3 +1,8 @@
 package env
 
-const OldEventTypePrefix = "sap.kyma.custom"
+type ActiveBackend string
+
+const (
+	JetStreamBackend = "JetStream"
+	EventMeshBackend = "EventMesh"
```

Comment:
nit: You can use the ActiveBackend typedef here directly :)

```suggestion
        JetStreamBackend ActiveBackend = "JetStream"
	EventMeshBackend ActiveBackend = "EventMesh"
```

Reply:
Shouldn’t this be inside `pkg/backend/jetstreamv2` instead ? It would then be `jetstreamv2.ActiveBackend` instead of `env.ActiveBackend`

Reply:
ideally, but this will be cleaned up after the deprecation of v1alpha1 code base.